### ISSUE TITLE
Match the Infrastructure details page design with Figma design

### ISSUE THREAD

**Issue by gorkem-bwl** [bluewave-labs/Checkmate#1291]
Currently there are a few missing decorations in the Infrastructure design page. 

Here is the Infra details page:

<img width="1119" alt="image" src="https://github.com/user-attachments/assets/78858926-0089-4811-b19b-76d17f42c0ab">

This is the [Figma design page](https://www.figma.com/design/RPSfaw66HjzSwzntKcgDUV/Uptime?node-id=2330-3449&t=cPnxnBs3UusfULfI-1): 

<img width="1060" alt="image" src="https://github.com/user-attachments/assets/ebaf385d-0ab8-43d5-b62f-d109a17ac9e4">

You need to fix fonts and add subtle background elements to the gauge elements.

Charts look fine, no need to touch them.

**Comment by pratikshelar546** [comment]
I would like to work on it

**Comment by gorkem-bwl** [comment]
Yep, go ahead @pratikshelar546 - make sure you share your final design before opening a PR for this. In case of doubt (e.g use of colors, fonts, components etc) please ask.

**Comment by pratikshelar546** [comment]
@gorkem-bwl sure started the work

**Comment by pratikshelar546** [comment]
is there any way to get dummy data

**Comment by gorkem-bwl** [comment]
> is there any way to get dummy data
> 
> i tried every possible way to get data and reach to this page but failed

You need to run the agent on a server to receive server side infrastructure data. I have necessary credentials and a secret for this. Please send an email to gorkem.cetin@bluewavelabs.ca so you can enter this information, and get real data from a server.

**Comment by pratikshelar546** [comment]
hello @ajhollid i am facing some issues to fix this issue. I'm not able to reach this page can you please help me into it and doing setup

**Comment by gorkem-bwl** [comment]
@pratikshelar546 can you write here what you did, step by step, what the output was, and where you are stuck? This is important to make sure Alex can pinpoint your problem.

**Comment by ajhollid** [comment]
> hello @ajhollid i am facing some issues to fix this issue. I'm not able to reach this page can you please help me into it and doing setup

Hi @pratikshelar546 ,

As @gorkem-bwl mentioned you need to run the [Capture agent](https://github.com/bluewave-labs/capture) on the target that you want to monitor. 

The easiest way to do this is just run the Capture agent on your local machine and point a monitor to `http://localhost:59323/api/v1/metrics`

The other options is to use our test server, you can reach out to the email address @gorkem-bwl provided above for credentials.

**Comment by gorkem-bwl** [comment]
Yep, I just provided @pratikshelar546 the credentials for our server so he can test it. 

@pratikshelar546 let me know if you are successful in connecting.

**Comment by pratikshelar546** [comment]
Hi @gorkem-bwl and @ajhollid,

Thank you both for your help!

@ajhollid, if you’re available and comfortable, could we connect on Slack to discuss the setup?

Apologies for taking up so much of your time with this!

**Comment by ajhollid** [comment]
Hi @pratikshelar546 ,

Walking you through the set up is beyond the scope of what I can help with, but if you have a specific issue with the setup that is causing a problem you can post it here and I can take a look.

The [developer setup documentation](https://bluewavelabs.gitbook.io/checkmate/users-guide/quickstart#dev-quickstart) is quite thorough and should provide all the information you need to get running.

**Comment by pratikshelar546** [comment]
Hi @ajhollid,

I followed all the steps mentioned in the document, and the app is running successfully. However, I encountered an issue:

I created an infrastructure to reach its details page, but the infrastructure's status remains pending. Since it’s not fully up, I am unable to access its details page.

Additionally, I noticed a mention about running the Capture agent on the target infrastructure that I want to monitor. Could you please confirm if this is a required step? If so, could you provide additional guidance on how to set it up?

Lastly, I received a URL from @gorkem-bwl. Could you clarify where exactly I need to use that URL in this process? I have already followed the steps mentioned in the Capture repository.

Thank you for your assistance!

**Comment by gorkem-bwl** [comment]
I'll leave the "pending" issue to Alex. 

> Lastly, I received a URL from @gorkem-bwl. Could you clarify where exactly I need to use that URL in this process? I have already followed the steps mentioned in the Capture repository.

In fact, you should already have done this from what I read in your message above. When you go to Infrastructure from the sidebar, you'll see nothing has been added yet. When you add the URL I gave you together with the secret, you should see the data flowing in from that server. You don't need to deploy Capture on another server/Linux docker instance. 

HTH.

**Comment by pratikshelar546** [comment]
@gorkem-bwl  Thank you for the clarification! The issue is now resolved, and I can see the data flowing in as expected. I can now focus on fixing the UI.

Thanks again for your help!

![image](https://github.com/user-attachments/assets/0e4d6826-7532-42e7-a533-1b256ccd45be)

**Comment by gorkem-bwl** [comment]
Yep! That's it!

**Comment by pratikshelar546** [comment]
Hi @gorkem-bwl,

I've updated the design. Could you please take a look and let me know if there are any changes that could further improve the UI?

![image](https://github.com/user-attachments/assets/4c720da1-8436-4a6f-bdc6-04df4245102e)

**Comment by gorkem-bwl** [comment]
Can you please also send the light theme? The dark theme is difficult to get an understanding of the UI details.

Is there an issue connecting to the server? It seems like the dashboard doesn't get data.

**Comment by pratikshelar546** [comment]
i followed all the steps that mention in document and put the details that you shared with me.
server is running with 0 errors in console.

![image](https://github.com/user-attachments/assets/af703e40-a532-4c83-ba53-774c340a78d7)

**Comment by gorkem-bwl** [comment]
I am sorry - it should be http://64.23.145.13:5001/api/v1/metrics (please add /metrics at the end).

**Comment by pratikshelar546** [comment]
I've updated the URL

Please check the UI in light mode and let me know if any adjustments are needed.


![image](https://github.com/user-attachments/assets/6c6976c3-fc41-4fe8-a14a-ec9b930e9975)

**Comment by gorkem-bwl** [comment]
Can you double check the Figma file again, as there are subtle background colors in "Used" / "Total" numbers?

**Comment by gorkem-bwl** [comment]
Those are the areas that require a modification. Basically in the cards, the font weights should be modified. Memory usage and CPU usage cards have a few more details like background and smaller fonts. Please make sure you check if those fonts are available in our font styles as well.

<img width="386" alt="image" src="https://github.com/user-attachments/assets/d4e3f216-e43f-4c0e-bba3-dc610e207530">

**Comment by pratikshelar546** [comment]
Sure I will do necessary changes and if I need help regarding font and colors I'll reach out here

**Comment by pratikshelar546** [comment]
hi @gorkem-bwl  i have done changes that you mentioned above.
![image](https://github.com/user-attachments/assets/e9a71702-cc55-4749-8fd5-09e91476e9d0)

**Comment by gorkem-bwl** [comment]
Can you double check colors with Figma? 

<img width="1327" alt="image" src="https://github.com/user-attachments/assets/21948f9e-2f15-47ef-960f-d1eec183fcc0">

<img width="1126" alt="image" src="https://github.com/user-attachments/assets/85244140-9078-4138-9887-0eecc7116595">

**Comment by gorkem-bwl** [comment]
Looks good! Thanks. Please send a PR.

**Comment by pratikshelar546** [comment]
I'd like to extend my gratitude to @gorkem-bwl and @ajhollid for their support. I'm excited to submit my first pull request to Checkmate and look forward to contributing more in the future.

asking one noob question i have to take pull from develop branch before creating pull request?

**Comment by gorkem-bwl** [comment]
> asking one noob question i have to take pull from develop branch before creating pull request?

Should be it. A core team member will respond if that's not the case.

**Comment by gorkem-bwl** [comment]
@pratikshelar546 confirmed that you need to pull from develop branch.

**Comment by pratikshelar546** [comment]
@gorkem-bwl Thank you for confirming. I have already taken a pull from the develop branch, there is no conflicts, pushed my changes, and successfully raised a PR. Please let me know if there's anything else required.

**Comment by Br0wnHammer** [comment]
Hey @gorkem-bwl, is this up for grabs?

**Comment by gorkem-bwl** [comment]
Yes and assigned. Please check previous PR as well.

**Comment by Br0wnHammer** [comment]
> Yes and assigned. Please check previous PR as well.

Sure. I will start as soon as my opened PR is merged.

**Comment by Br0wnHammer** [comment]
<img width="1277" alt="Image" src="https://github.com/user-attachments/assets/c057ca3e-97ff-40c1-9728-b1ff82e17fe3" />

<img width="644" alt="Image" src="https://github.com/user-attachments/assets/8b820734-444e-43d7-8983-ff17299de054" />

@gorkem-bwl, while improving on the design I can also fix a typo in the code, as shown in the screenshot.

**Comment by gorkem-bwl** [comment]
@Br0wnHammer please do. Thank you. I got your email btw, and send you a response about that soon.

**Comment by gorkemcetin** [comment]
> [@Br0wnHammer](https://github.com/Br0wnHammer) please do. Thank you. I got your email btw, and send you a response about that soon.

Sent you credentials that you can use while fixing the UI.

**Comment by Br0wnHammer** [comment]
> > [@Br0wnHammer](https://github.com/Br0wnHammer) please do. Thank you. I got your email btw, and send you a response about that soon.
> 
> Sent you credentials that you can use while fixing the UI.

Received, I will send a UI screenshot here first before opening a PR.

**Comment by gorkem-bwl** [comment]
Yea, that will make things easier for me really. Thanks.

**Comment by Br0wnHammer** [comment]
<img width="1271" alt="Image" src="https://github.com/user-attachments/assets/d88f4af5-c426-4709-8211-076ecb432524" />

Hey @gorkem-bwl, thoughts?

**Comment by gorkem-bwl** [comment]
Looks good to me so far. It lacks the following two UI aspects (the subtle bg inside the rectangle, and the bg around the circle):

![Image](https://github.com/user-attachments/assets/3eb86937-9989-4685-9f91-dc351ba8384d)

Also can you decrease the gradient just a bit here? It's always hurted my eyes :)

<img width="177" alt="Image" src="https://github.com/user-attachments/assets/7d12fabf-847f-4bc6-9871-94c32f8cfcd8" />

> The only color I have added to your theme is the gauge base color : lightBlueWave: "#CDE2FF",

OK, makes sense. Please open a PR.

**Comment by Br0wnHammer** [comment]
Sure, let me make the said changes are share a UI. Afterwards, I will open the PR.

**Comment by Br0wnHammer** [comment]
<img width="818" alt="Image" src="https://github.com/user-attachments/assets/da6aafed-6376-4c7f-acaa-a1f6961bbeb3" />

@gorkem-bwl, is this color fine for dark mode?

<img width="262" alt="Image" src="https://github.com/user-attachments/assets/57d02551-488a-443f-ad26-6ac4b1433d16" />

This is how it looks in light mode.

**Comment by gorkem-bwl** [comment]
Yes, both look fine. In light mode I barely sense the background color, but probably it's due to the fact that Github downscaled the colors. How does it look on your monitor?

**Comment by Br0wnHammer** [comment]
> Yes, both look fine. In light mode I barely sense the background color, but probably it's due to the fact that Github downscaled the colors. How does it look on your monitor?

Looks fine here, The color stated in the figma file was:#FCFCFC
While I used the pre-defined one which are basically the same:#FCFCFD

**Comment by Br0wnHammer** [comment]
Gorkem, regarding the gradient modification. The same setting has been applied to all the status boxes. Do you want me to modify all of them to tone it down a bit?

**Comment by gorkem-bwl** [comment]
Can you elaborate a bit? If I am not mistaken, this is a component, so once there is a change, it should be applied to all status boxes (in uptime section, for example).

**Comment by Br0wnHammer** [comment]
> Can you elaborate a bit? If I am not mistaken, this is a component, so once there is a change, it should be applied to all status boxes (in uptime section, for example).

Yes, precisely what I am trying to say. I was confirming that all the gradients of such stat boxes need to be tone downed.

**Comment by Br0wnHammer** [comment]
<img width="842" alt="Image" src="https://github.com/user-attachments/assets/ba600e07-af01-444a-94c9-5f78b9b3f914" />

@gorkem-bwl , is this better?

**Comment by gorkem-bwl** [comment]
Yes, this is easy to the eyes.

**Comment by Br0wnHammer** [comment]
Hello Gorkem, this issue can now be closed as I do not have the necessary permissions.
